docs/Linking.md

```diff
@@ -2,6 +2,27 @@
 
 ## iOS
 
+#### Project linking
+1. Open your project `.xcodeproj` on xcode.
+
+2. Right click on the Libraries folder and select `Add files to "yourProjectName"`.
+
+3. Add `RNCAsyncStorage.xcodeproj` to your project Libraries.
```

Comment:
Mind adding where we can find that `.xcodeproj` file? 
Location: `node_modules/@react-native-community/async-storage/ios`

Reply:
Also done 💪